.github/workflows/ci.yaml

```
@@ -13,7 +13,7 @@ jobs:
     env:
       REPLICA: 5
       BEE_URL: 'http://bee-0.localhost'
-      BEE_CHEQUEBOOK_URL: 'http://bee-0-debug.localhost'
+      BEE_TEST_CHEQUEBOOK: 'test'
```

Any reason this is not just `true`?
```suggestion
      BEE_TEST_CHEQUEBOOK: true
```